static/panes/gccdump-view.ts

```diff
@@ -1,5 +1,4 @@
-// Copyright (c) 2017, Marc Poulhiès - Kalray Inc.
-// Copyright (c) 2022, Compiler Explorer Authors
+// Copyright (c) 2017, Marc Poulhiès - Kalray Inc, 2022, Compiler Explorer Authors
```

Comment:
With the old way, the linter was complaining about a bad header. Is this format ok?

Reply:
I've never seen it like this. The lint wants a unique copyright notice?

Reply:
The pattern it's looking for is `^ Copyright \\(c\\) \\d{4}, .*$` which I'd think works for two headers, but somehow it must be making sure that the template is also being followed, which trips the linter here. No idea what to do